TUTORIAL.md

```diff
@@ -0,0 +1,33 @@
+# Guide to Solving Operators
+
+### `unsafe?`
+
+After running `rspec` for the first time we see we have seven tests failing.
+
```

Comment:
This is super early in the semester. Can you show the failing tests and then explain them a bit? Like how did you go from failing test to that method below?